src/app/components/Metadata/index.tsx

```diff
@@ -122,6 +128,11 @@ const MetadataContainer = ({
     ...(isAmp && { amp: '' }), // empty value as this makes Helmet render 'amp' as per https://www.ampproject.org/docs/fundamentals/spec#ampd
   };
 
+  const { liteSite = defaultTranslations } = translations;
+  const { dataSaving } = liteSite;
+
+  const showLiteTitle = isLite && liteEnabledServices.includes(service);
+  const litePageTitle = `${title} - ${dataSaving}: ${brandName}`;
```

Comment:
Do we need/have a test to ensure dataSaving is there?

Reply:
Yep here - https://github.com/bbc/simorgh/blob/cd2dad45687a93ce4d6e2880ba1070ceecf4e724/src/app/components/Metadata/index.test.tsx#L177-L187